nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -187,6 +187,8 @@
 
 - The `replay-sorcery` package and module was removed as it unmaintained upstream. Consider using `gpu-screen-recorder` or `obs-studio` instead.
 
+- The `services.samba.extraConfig` setting has been removed and converted to the freeform type option named [`services.samba.settings`](#opt-services.samba.settings) according to RFC42.
```

Comment:
Please also mention the removal of `configText`.

And for completeness it might also be worth to mentioned the renamed options here.

Reply:
I modified rl-2411.section.md to mention that configText has been removed in favor of services.samba.settings freeform type option.

And also added the renamed options here.